pennylane_qiskit/vqe/vqe_runner.py

```diff
@@ -0,0 +1,548 @@
+            if len(tape.operations) == 0:
+                raise qml.QuantumFunctionError("Function contains no quantum operation")
+
+            # if no wire ordering is specified, take wire list from tape
+            wires = tape.wires
+
+            # Compare the number of qubits from the circuit and from the hamiltonian
+            num_qubits_c = len(wires)
+
+            if num_qubits_c > num_qubits_h:
+                num_qubits = num_qubits_c
+            elif num_qubits_c <= num_qubits_h:
+                num_qubits = num_qubits_h
+
+            consecutive_wires = qml.wires.Wires(range(num_qubits))
+            wires_map = OrderedDict(zip(wires, consecutive_wires))
+
+            # From here: Create the Qiskit ansatz circuit
+            params_vector = ParameterVector("p", num_params)
+
+            reg = QuantumRegister(num_qubits, "q")
+            circuit_ansatz = QuantumCircuit(reg, name="vqe")
+
+            circuits = []
+
+            j = 0
+            for operation in tape.operations:
+                wires = qml.wires.Wires([wires_map[wire] for wire in operation.wires.tolist()])
+                par = operation.parameters
+                operation = operation.name
+                mapped_operation = QiskitDevice._operation_map[operation]
+
+                qregs = [reg[i] for i in wires.labels]
+
+                if operation.split(".inv")[0] in ("QubitUnitary", "QubitStateVector"):
+                    # Need to revert the order of the quantum registers used in
+                    # Qiskit such that it matches the PennyLane ordering
+                    qregs = list(reversed(qregs))
+
+                dag = circuit_to_dag(QuantumCircuit(reg, name=""))
+
+                if operation in ("QubitUnitary", "QubitStateVector"):
+                    gate = mapped_operation(par[0])
+                else:
+                    if par:
+                        op_num_params = len(par)
+                        par = []
+                        for num in range(0, op_num_params):
+                            par.append(params_vector[j + num])
+                        j += op_num_params
+                        gate = mapped_operation(*par)
+                    else:
+                        gate = mapped_operation
+
+                if operation.endswith(".inv"):
+                    gate = gate.inverse()
+
+                dag.apply_operation_back(gate, qargs=qregs)
+                circuit = dag_to_circuit(dag)
+                circuits.append(circuit)
+
+            for circuit in circuits:
+                circuit_ansatz &= circuit
+
+            inputs["ansatz"] = circuit_ansatz
+
+        else:
+            raise ValueError("Input ansatz is not a tape, quantum function or a str")
+
+    # The circuit will be taken from the Qiskit library as a str was passed.
+    elif isinstance(ansatz, str):
+
+        num_qubits = num_qubits_h
+
+        ansatz_circ = getattr(lib_local, ansatz, None)
+        if not ansatz_circ:
```

Comment:
Minor: `if not ansatz_circ` could evaluate to `True` not just for `None`.
```suggestion
        if ansatz_circ is None:
```